soweego/linker/classify.py

```diff
@@ -130,10 +123,13 @@ def _zero_when_different_names(prediction, wikidata, target):
 
 
 def _add_missing_feature_columns(classifier, feature_vectors):
+
     if isinstance(classifier, rl.NaiveBayesClassifier):
         expected_features = len(classifier.kernel._binarizers)
-    elif isinstance(classifier, rl.SVMClassifier):
+    
+    elif isinstance(classifier, classifiers.SVCClassifier):
```

Comment:
```suggestion
    elif isinstance(classifier, (classifiers.SVCClassifier, rl.SVMClassifier)):
```
Keeping support for LinearSVC.